### DISCUSSION TITLE
🚨 New Reporting System

### DISCUSSION THREAD

**Discussion by berrysauce** [VictiniX888/Illegal-Mod-Sites#537] [General]
We have revamped our reporting system. Share any feedback or questions here, and we'll try to answer ASAP.

**Comment by Brandonbr1** [comment]
Wait so when someone submits with this new system only project maintainers can see it? And no one else can see it or give their own opinion on it via comments

**Reply by Brandonbr1** [reply]
> Wait so when someone submits with this new system only project maintainers can see it?

I do not like this new system so far.

**Reply by berrysauce** [reply]
Yes, you can't see the individual reports, but you can see our general progress here: https://report.stopmodreposts.org/pages/progress.

It's practical since this system doesn't allow double submissions and makes it easier for us to review sites.

We'll open issues on GitHub if we find controversial submissions.

Thanks for your feedback though!

**Reply by Brandonbr1** [reply]
> Yes, you can't see the individual reports, but you can see our general progress here: https://report.stopmodreposts.org/pages/progress.
> 
> It's practical since this system doesn't allow double submissions and makes it easier for us to review sites.
> 
> We'll open issues on GitHub if we find controversial submissions.
> 
> Thanks for your feedback though!

This is just my feedback on this new system 😃

Ok  I get why you did this but:

> practical since this system doesn't allow double submissions
>

Couldn't you make a GitHub bot that closes all double submissions?

Edit:fix grammar to make me not sound angry

**Reply by berrysauce** [reply]
I know, I wasn't criticising your feedback, just answering it.

Theoretically, but we now have a database of submissions instead of a list of issues, and we have an application to streamline the review process. I get your concern, but it's a price to pay for optimisation.
